plugins/3MFWriter/ThreeMFWriter.py

```diff
@@ -15,7 +15,7 @@
 
 from PyQt6.QtCore import QBuffer
 
-import Savitar
+import pySavitar
```

Comment:
```suggestion
import pySavitar as Savitar 
```

That's a much easier way to do things; No other code needs to change that way.